.travis.yml

```diff
@@ -5,7 +5,9 @@ node_js:
   - "0.12"
   - "0.11"
   - "0.10"
-  - "4.0"
+  - "4.0",
```

Comment:
Only test on `10`, `8` and `6`. Drop all the others as I'm not interested in them anymore, they are all EOL.

Reply:
Sounds like a plan... will do.